### ISSUE TITLE
Make deleted accounts visible to moderators

### ISSUE THREAD

**Issue by RicoElectrico** [openstreetmap/openstreetmap-website#3585]
### Description

Hi,
There is a vandal that - as means of evasion and obfuscation - deletes his sock-puppet accounts after receiving a permanent ban, which makes it hard to track of changesets and discussions under them (Pascal Neis' site won't show discussions on deleted users' changsesets).

It'd be useful in the future for mods to be able to browse such profiles.

Example:
https://www.openstreetmap.org/user_blocks/4958
https://www.openstreetmap.org/user_blocks/5315
https://www.openstreetmap.org/user_blocks/5302
https://www.openstreetmap.org/user_blocks/5301
https://www.openstreetmap.org/user_blocks/5300
https://www.openstreetmap.org/user_blocks/5264
https://www.openstreetmap.org/user_blocks/5263
https://www.openstreetmap.org/user_blocks/5262
https://www.openstreetmap.org/user_blocks/4876
https://www.openstreetmap.org/user_blocks/5849
https://www.openstreetmap.org/user_blocks/6110

**Comment by mmd-osm** [comment]
Is this issue still relevant now that we have a 7 day cool down period, in which a user cannot delete their account after their last edit?

**Comment by SomeoneElseOSM** [comment]
> Is this issue still relevant now that we have a 7 day cool down period, in which a user cannot delete their account after their last edit?

Yes, just to take the first example on the list above, if I want to investigate edits by the user blocked in https://www.openstreetmap.org/user_blocks/4958 it's still a pain.

Also, what version is live?  https://github.com/openstreetmap/openstreetmap-website/blob/master/config/settings.yml#L59 is null.

**Comment by mmd-osm** [comment]
It's in chef, and 7 *24 hours: https://github.com/openstreetmap/chef/blob/7b5bc04fbe3cd59686c9660aae8a66b890e27724/cookbooks/web/recipes/rails.rb#L151
